### DISCUSSION TITLE
Argon HP Bar gets massively resized after skin is reloaded

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#25429] [Q&A]
Before:
![osu_2023-11-13_23-17-09](https://github.com/ppy/osu/assets/74088315/11d5b56f-387d-4185-b8de-76a41ccd7705)

After (simply switching to another skin and then back to this one):
![osu_2023-11-13_23-17-20](https://github.com/ppy/osu/assets/74088315/2f7ab404-1a69-4f1f-bb86-f54928d02893)

**Comment by WalmartWhiteCat** [comment]
I also found that immediately. I hope it's an easy fix
